src/test/java/com/alipay/oceanbase/rpc/ObTableClientFilterColumnTest.java

```diff
@@ -0,0 +1,335 @@
+            Assert.assertEquals(null, obtableAggregationResultWithFilter.get("max(c2)"));
+            Assert.assertEquals(null, obtableAggregationResultWithFilter.get("min(c2)"));
+            Assert.assertEquals(null, obtableAggregationResultWithFilter.get("count()"));
+            Assert.assertEquals(null, obtableAggregationResultWithFilter.get("sum(c2)"));
+            Assert.assertEquals(null, obtableAggregationResultWithFilter.get("avg(c2)"));
+
+        } finally {
+        }
+    }
+
+    /*
+    CREATE TABLE test_aggregation (
+    `c1` varchar(255),
+    `c2` int NOT NULL,
+    `c3` bigint NOT NULL,
+    `c4` float NOT NULL,
+    `c5` double NOT NULL,
+    `c6` tinyint NULL,
+    `c7` datetime,
+    PRIMARY KEY(`c1`)
+    );
+     */
+
+    @Test
+    // Test aggregation with big int , should report error out of range
+    public void testAggregationWithBigint() throws Exception {
+        final String TABLE_NAME = "test_aggregation";
+        try{
+            final ObTableClient client = (ObTableClient) this.client;
+            client.addRowKeyElement(TABLE_NAME, new String[]{"c1"});
+            SimpleDateFormat sdf = new SimpleDateFormat(" yyyy-MM-dd HH:mm:ss ");
+            Date date1 = sdf.parse(" 1000-12-10 19:20:00 ");
+            Date date2 = sdf.parse(" 1010-07-10 19:20:00 ");
+            Date date3 = sdf.parse(" 1100-02-10 19:20:00 ");
+
+            client.insert(TABLE_NAME, "first_row", new String[]{"c2", "c3", "c4", "c5",
+                    "c6", "c7"}, new Object[]{1, 9223372036854775807L, 1.0f, 1.0, (byte) 1, date1});
+            client.insert(TABLE_NAME, "second_row", new String[]{"c2", "c3", "c4", "c5",
+                    "c6", "c7"}, new Object[]{2, 9223372036854775807L, 2.0f, 2.0, (byte) 2, date2});
+            client.insert(TABLE_NAME, "third_row", new String[]{"c2", "c3", "c4", "c5",
+                    "c6", "c7"}, new Object[]{3, 9223372036854775807L, 3.0f, 3.0, (byte) 3, date3});
+            ObTableAggregation obtableAggregation = client.aggregate(TABLE_NAME);
+
+            // test
+            obtableAggregation.sum("c3");
+
+            ObTableAggregationResult obtableAggregationResult = obtableAggregation.execute();
+
+            Assert.assertEquals(10L, obtableAggregationResult.get("sum(c3)"));
+
+        } catch (Exception e) {
+            e.printStackTrace();
+            Assert.assertEquals("bigint value out of range",
+                    ((NullPointerException) e).getMessage());
+        } finally {
+        }
+    }
+
+    /**
+    table :
+    create table cse_index_1 (
+     measurement VARBINARY(1024) NOT NULL,
+     tag_key VARBINARY(1024) NOT NULL,
+     tag_value VARBINARY(1024) NOT NULL,
+     series_ids MEDIUMBLOB NOT NULL,
+     PRIMARY KEY(measurement, tag_key, tag_value))
+    partition by key(measurement) partitions 13;
+    **/
+
+    @Test
+    public void testBatchInsertJudge() throws Exception {
```

Comment:
Move it into ObTableClientTest.
BTW, what do you want to test in this test? If you plan to test Medium Blob, you should refactor your test name into testBatchOpsWithMedumBlob or something related to your purpose.

Reply:
already edit